packages/core/admin/admin/src/components/ThemeToggleProvider.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+
+import { DefaultTheme } from 'styled-components';
+
+import { ThemeToggleContext } from '../contexts/themeToggle';
+
+const THEME_KEY = 'STRAPI_THEME';
+
+const getDefaultTheme = () => {
+  const browserTheme = window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light';
+  const persistedTheme = localStorage.getItem(THEME_KEY) as 'dark' | 'light' | null;
```

Comment:
I'm wondering if we should centralize the type for theme names, because we are repeating it at least 3 times already?

Reply:
Yeah i think this is the clash between your work and mine and i've not cleaned my stuff up enough, let me take another look 🖖🏼